donate.html

```diff
@@ -248,6 +248,23 @@ <h5 class="margin-bottom-normal ak-donate-three-step-hidden">Amount:</h5>
                               onkeypress="return (event.charCode == 8 || event.charCode == 0) ? null : event.charCode >= 48 && event.charCode <= 57"
                               id="ak-other-amount-field" class="text-font-display" name="amount_other" size="3">
                           </label>
+                          <script>

```

Comment:
event documentation for reference:
https://developers.google.com/analytics/devguides/collection/ua/gtm/enhanced-ecommerce#product-impressions